tests/host.cc

```diff
@@ -0,0 +1,34 @@
+#include <clap/helpers/plugin-proxy.hxx>
+#include <clap/helpers/host.hh>
+#include <clap/helpers/host.hxx>
+
+#include <catch2/catch_all.hpp>
+
+template class clap::helpers::Host<clap::helpers::MisbehaviourHandler::Terminate,
```

Comment:
Maybe we want to use a macros to expand all the combinations?

Reply:
good idea, maybe in a later PR?